src/Marten.Testing/Harness/OneOffConfigurationsContext.cs

```diff
@@ -0,0 +1,120 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using Baseline;
+using Marten.Testing.Events;
+using Xunit;
+
+namespace Marten.Testing.Harness
+{
+    /// <summary>
+    /// Use this if the tests in a fixture are going to use
+    /// all custom StoreOptions configuration
+    /// </summary>
+    public abstract class OneOffConfigurationsContext
```

Comment:
**SUGGESTION:** Maybe it would be more explanatory to name it "CustomDocumentStoreConfigurationsContext"?

Reply:
I'd rather not tbh.